.github/CHANGELOG.md

```diff
@@ -14,6 +14,13 @@
   in `_device.py`.
   [#374](https://github.com/XanaduAI/pennylane/pull/374)
 
+* The `QNode` class is split into a hierarchy of simpler classes.
+  [#354](https://github.com/XanaduAI/pennylane/pull/354)
```

Comment:
Is this "official" yet? Or is it still shadowing the original ``QNode`` class?

Reply:
PL still uses the old QNode.

Reply:
Yep, still shadowing. Perhaps with each new QNode PR, we just add the link to the PR to the same changelog entry.